hackage2nix/Main.hs

```diff
@@ -153,6 +153,7 @@ main = do
                   & metaSection.hydraPlatforms %~ (`Set.difference` Map.findWithDefault Set.empty name (dontDistributePackages config))
                   & metaSection.maintainers .~ Map.findWithDefault Set.empty name globalPackageMaintainers
                   & metaSection.hydraPlatforms %~ (if isInDefaultPackageSet then id else const Set.empty)
+                  & metaSection.homepage .~ ("https://hackage.haskell.org/package/" ++ display pkgId)
```

Comment:
Please set `homepage` to an empty string `""` instead to suppress the `meta.homepage` outout altogether. That way, we can let the generic builder construct the appropriate URL from its default settings rather than repeating an almost identical Hackage URL several thousand times in the generated file.